src/udp_packets.rs

```diff
@@ -14,7 +14,7 @@ pub mod udp_payload {
     /// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
     /// |            Length             |           Checksum            |
     /// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-    /// We append the data, based on the port the the header
+    /// We append the data, based on the port the header
```

Comment:
Although I agree with the change, does this sentence make sense? 

> _"…based on the port the header"_

Reply:
Ah, should this be "the port **in** the header"?

Reply:
"We append the data, based on the port"

Reply:
As in that should be the complete sentence?

I think that makes it a bit ambiguous: it's below a diagram with both "Source Port" and "Destination Port"—to which one is it referring? Or neither?

I think it'd be useful to clarify that as part of this, if possible…?

Reply:
Ahh then it should be "We append the data, based on the destination port"